src/marketplace/MakeAComponentSaleView.js

```diff
@@ -1,33 +1,35 @@
 //@flow strict
 import * as React from 'react';
 import Cookies from 'universal-cookie';
+import { ToastContainer ,toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.min.css';
 
 type Props = {||}; 
 type State = {|
-    userId: string,
-    salePrice: number,
-    itemID: string,
-    itemAmount: number,
+	userId: string,
+	salePrice: number,
+	itemID: string,
+	itemAmount: number,
 |};
 let itemsToSell = [{value: '', label: ''}];
 class MakeAComponentSaleView extends React.Component<Props, State> {
 
-    
-    constructor() {
-        super();
-        this.state={
-            userId: '',
-            salePrice: 0,
-            itemID: '',
-            itemAmount: 0,
-        }
-        this.getUserInventory();
-    }
+
+	constructor() {
+		super();
+		this.state={
+			userId: '',
+			salePrice: 0,
+			itemID: '',
+			itemAmount: 0,
+		}
+		this.getUserInventory();
+	}
 
 
-    getUserInventory = async ():Promise<void> => {
-        itemsToSell = [];
-        itemsToSell.push({value: '', label: ''});
+	getUserInventory = async ():Promise<void> => {
+		itemsToSell = [];
```

Comment:
What's this? https://www.youtube.com/watch?v=QLvvkTbHjHI

(Array of what type?)

Reply:
Yeah this is just a toasty review these changes are coming when I handle the infinite request which I grouped together with issue 294

Reply:
okay, sounds good